### PR TITLE
Allow for tree-sitter 0.20 and above

### PR DESCRIPTION
**Pull request by note89**
I am using this package together with quite a few other tree-sitter libs.

This line makes it so this package cannot be used with other tree-sitter-<lang> packages that want 0.21 or newer, like
https://github.com/tree-sitter/tree-sitter-rust/blob/master/Cargo.toml#L23
https://github.com/tree-sitter/tree-sitter-cpp/blob/master/Cargo.toml#L23
https://github.com/tree-sitter/tree-sitter-c/blob/master/Cargo.toml#L23
https://github.com/tree-sitter/tree-sitter-php/blob/master/Cargo.toml#L31
https://github.com/tree-sitter/tree-sitter-go/blob/master/Cargo.toml#L23
https://github.com/tree-sitter/tree-sitter-python/blob/master/Cargo.toml#L24

Can we bump this?

### CODE REVIEW

**Comment by JoranHonig** [comment]
Would things work if we do `>= 0.20.0`?  

I believe that's necessary to maintain compatibility with tree_sitter_graph

**Comment by JoranHonig** [comment]
@note89 thanks for your contribution!

tree-sitter seems to have introduced a new way to go about dependencies, using tree-sitter-language as a dependency in the grammar instead of depending on tree-sitter.

tl;dr I published a new version of the package using this new dependency & you should be able to use your preferred tree-sitter version!
